package.json

```diff
@@ -23,28 +28,64 @@
     "schema",
     "parser"
   ],
-  "author": "Fran Mendez (fmvilas.com)",
```

Comment:
why did you remove it? Fran was/is the author of this package

Reply:
We usually add this:

```json
  "author": {
    "name": "The AsyncAPI maintainers",
    "url": "https://www.asyncapi.com"
  },
```

Reply:
> why did you remove it? Fran was/is the author of this package

Removed it accidentally because of copy-paste party. @derberg  Should I just leave @fmvilas or rather add what @magicmatatjahu suggested?

Reply:
well, I say leave it as it was originally, as it reflects reality 😄 he created it, and if we really think is is needed, we can add maintainers into `contributors` array. Anyway, not really related to this PR

Reply:
Reverted https://github.com/asyncapi/avro-schema-parser/pull/168/files#r1006720043